stravalib/unit_registry.py

```diff
@@ -0,0 +1,4 @@
+from pint import UnitRegistry
+
+ureg = UnitRegistry()
+Q_ = ureg.Quantity
```

Comment:
This needs to be in a separate module to prevent potential multiple imports (and as such multiple unit registries)

Reply:
wow - that's interesting actually but makes sense that you only want to instantiate the registry object once.